docs/manual/cluster-admin/alerting-and-troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+
+## Set Up Alert Manager
+
+OpenPAI's alert manager is set to send alerting e-mails when alert happens. To begin with, you should get an SMTP account to send these e-mails.
+
+After getting an SMTP account, how to set up the alert manager in PAI? Please read the document about [service management and paictl](./basic-management-operations.md#pai-service-management-and-paictl) first, and start a dev box container. Then, in the dev box container, pull the configuration by:
+
+```bash
+./paictl config pull -o /cluster-configuration
+```
+
+Uncomment the alert manager section in `/cluster-configuration/services-configuration.yaml`, and set your SMTP account and the receiver's e-mail address. Here is an example:
+
+```bash
+alert-manager:
+  port: 9093
+  receiver: <receiver-email-address>
+  smtp_auth_password: <smtp-password>
+  smtp_auth_username: <smtp-username>
+  smtp_from: <smtp-email-address>
+  smtp_url: <smtp-server>:<smtp-port>
+```
+
+Configuration `port` stands for the port of alert manager. In most cases, you don't need to change it. Configuration `receiver` is usually set to be the administrator's e-mail address to receive alerting e-mails.
+
+Save the configuration file, and start alert manager by:
+
+```bash
+./paictl.py service start -n alert-manager
+./paictl.py config push -p /cluster-configuration -m service
+./paictl.py service start -n alert-manager
+```
+
+After alert manager is successfully started, the receiver's e-mail address will receive alerting e-mails from the SMTP account. Also, you can view the alerting information on Webportal (in the top-right corner):
+
+   <img src="./imgs/alert-on-webportal.png" width="100%" height="100%" />
+
+
+## Troubleshooting
+
+### PaiServicePodNotReady Alert
+
+This is a kind of alert from alert manager, and usually caused by container being killed by operator or OOM killer. To check if it was killed by OOM killer, you can check node's free memory via Prometheus:
+
+  1. visit Prometheus web page, it is usually `http://<your-pai-master-ip>:9091`.
+  2. Enter query `node_memory_MemFree_bytes`.
+  3. If free memory drop to near 0, the container should be killed by OOM killer
+  4. You can double check this by logging into node and run command `dmesg` and looking for phase `oom`.
+
+Solutions:
+
+  1. Force remove unhealth containers with this command in terminal:
+  `kubectl delete pod pod-name --grace-period=0 --force`
+  2. Recreate pod in Kubernetes, this operation may block indefinitely because dockerd may not functioning correctly after OOM. If recreate blocked too long, you can log into the node and restart dockerd via `/etc/init.d/docker restart`.
```

Comment:
Should we increase pod memory limit for this case?

Reply:
Added the following:

```
3. If restarting doesn't solve it, you can consider increase the pod's memory limit.
```